app/controllers/operations_controller.rb

```diff
@@ -0,0 +1,55 @@
+class OperationsController < ApplicationController
+  # before_action :set_operation, only: %i[ show edit update destroy ]
+  before_action :authenticate_user!
+  before_action :set_client
+  # GET /operations or /operations.json
+  def index
+    @group = Group.find(params[:group_id])
+    @operations = @group.operations
+  end
+
+  # GET /operations/1 or /operations/1.json
+  # def show
+  # end
+
+  # GET /operations/new
+  def new
+    @operation = Operation.new
+  end
+
+  # GET /operations/1/edit
+  # def edit
+  # end
```

Comment:
- [OPTIONAL] Please always make sure you remove commented code from your codebase, as this makes your codebase cleaner.